.github/workflows/main.yml

```diff
@@ -57,56 +57,27 @@ jobs:
         id: cache-npm
         uses: actions/cache@v3
         with:
-          path: ~/.npm
-          key: |
-            modules-${{ hashFiles('**/package-lock.json') }}
-
-      - name: Install dependencies
-        if: steps.cache.outputs.cache-npm != 'true'
-        run: cd UI && npm ci --ignore-scripts
-
-      - name: Cache node modules
-        id: cache-node_modules
-        uses: actions/cache@v3
-        with:
-          path: ~/.npm
+          path: |
+             ~/.npm
+             UI/node_modules
```

Comment:
that action is superseeded by `actions/setup-node`. I'd like to cache node_modules, because it skips a step in setup: even though yarn is *fast*, we can even just do nothing (which will even outperform yarn :-) )